src/renderer/screens/workspace-screen.tsx

```
@@ -1115,396 +1127,337 @@ const WorkspaceScreen = () => {
+          const fbContextKey =
+            currentPou.type === 'function-block' ? fbSelectedInstance.get(currentPou.data.name.toUpperCase()) ?? '' : ''
+
+          const cached = diagramVarKeysCache.current
+          if (
+            cached &&
+            cached.pouName === pouName &&
+            cached.language === pouLanguage &&
+            cached.fbContextKey === fbContextKey
+          ) {
+            // Reuse cached keys
+            cached.keys.forEach((key) => debugVariableKeys.add(key))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)